smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/GetTogglesIT.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019 Adobe Systems Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adobe.cq.cloud.testing.it.smoke;
+
+import com.adobe.cq.testing.client.CQClient;
+import com.adobe.cq.testing.junit.rules.CQAuthorPublishClassRule;
+import com.adobe.cq.testing.junit.rules.CQRule;
+import com.adobe.cq.testing.junit.rules.Page;
+
+import com.google.gson.Gson;
```

Comment:
Gson is a transient dependency, but it might be removed at some point, so please use fasterxml, as detailed here: https://github.com/adobe/aem-testing-clients/wiki/Best-practices#use-fasterxml-jackson-library-for-handling-jsons